openqabot/loader/gitea.py

```diff
@@ -155,22 +155,29 @@ def add_build_result(
     failed_packages: Set[str],
 ):
     state = res.get("state")
-    if state != "published":
-        unpublished_repos.add("@".join([res.get("project"), res.get("arch")]))
-        return
+    project = res.get("project")
     for scminfo_element in res.findall("scminfo"):
         found_scminfo = scminfo_element.text
-        existing_scminfo = incident.get("scminfo", None)
+        project_match = re.search(".*:PullRequest:\\d+:(.*)", project)
+        if project_match:
+            scm_info_key = "scminfo_" + project_match.group(1)
+        else:
+            scm_info_key = "scminfo"
```

Comment:
```suggestion
        scm_info_key = "scminfo_" + project_match.group(1) if project_match else "scminfo"
```

this way it's clear that the scm_info_key is always set, just to different values